libs/prelude/Prelude.idr

```diff
@@ -214,6 +214,30 @@ Enum Char where
 
   pred c = fromNat (pred (toNat c))
 
+Enum Bits8 where
+  toNat b   = fromIntegerNat (prim__zextB8_BigInt b)
```

Comment:
Actually, have you considered just making the instance work with `Bits n` type in `Data.Bits`?
Maybe that is easier/more useful, but I will leave it up to you.